### PR TITLE
Fix code style

### PR DESCRIPTION
**Pull request by iam4x**
* add a command `$ npm run lint` for linting lib files
* correct code style of files

### CODE REVIEW

**Comment by hd-deman** [comment]
:+1:

**Comment by steelbrain** [comment]
Looks good to me.

**Comment by iam4x** [comment]
Let's merge so :+1:

File: lib/panel.coffee [hunk 1]
```diff
@@ -17,7 +19,8 @@ class Panel
       return unless Message.Position
       return if @Type is 'file' and (not Message.CurrentFile)
       P = Message.Position
-      Marker = @Linter.ActiveEditor.markBufferRange [[P[0][0] - 1, P[0][1] - 1], [P[1][0] - 1, P[1][1]]], {invalidate: 'never'}
+      range = [[P[0][0] - 1, P[0][1] - 1], [P[1][0] - 1, P[1][1]]]
```
Comment (steelbrain) [inline lib/panel.coffee]: `r` is lowercase :P

Comment (iam4x) [inline lib/panel.coffee]: `Range` is a constructor already defined, we could have issues with coffeescript variables scope.

Comment (steelbrain) [inline lib/panel.coffee]: It's not defined in this file, and coffeescript doesn't have any magical powers, so It would work.

**Pull request merged**